compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```
@@ -4371,6 +4371,22 @@
                     "type": "ExpressionNode"
                 }
             ]
+        },
+        {
+            "name": "TupleMemberDescriptorNode",
+            "base": "TypeDescriptorNode",
+            "kind": "TYPE_DESC",
+            "attributes": [
+                {
+                    "name": "annotation",
```

Similar to we have defined in other places
```suggestion
                    "name": "annotations",
```